pyaedt/mechanical.py

```diff
@@ -632,23 +644,23 @@ def assign_heat_generation(self, objects_list, value, boundary_name=""):
             return bound
         return False
 
-    @pyaedt_function_handler()
-    def create_setup(self, setupname="MySetupAuto", setuptype=None, **kwargs):
+    @pyaedt_function_handler(setupname="name", setuptype="setup_type")
+    def create_setup(self, name="MySetupAuto", setup_type=None, **kwargs):
         """Create an analysis setup for Mechanical.
 
-        Optional arguments are passed along with ``setuptype`` and ``setupname``.  Keyword
+        Optional arguments are passed along with ``setup_type`` and ``name``.  Keyword
         names correspond to the ``setuptype``
```

Comment:
I keep noticing the use of both `setup_type`` and ``setuptype``. Is this intentional?

Reply:
We should just keep setup_type